libs/community/langchain_community/document_loaders/mongodb.py

```diff
@@ -20,13 +20,37 @@ def __init__(
         *,
         filter_criteria: Optional[Dict] = None,
         field_names: Optional[Sequence[str]] = None,
+        metadata_names: Optional[Sequence[str]] = None,
+        include_db_collection_in_metadata: bool = False,
```

Comment:
should this default to True, to avoid breaking changes for users?

Reply:
Yeah, You're right. I've changed the default value to True. Thanks